.prettierrc

```diff
@@ -0,0 +1,18 @@
+{
+  "arrowParens": "avoid",
```

Comment:
is this meant to be "Parents"?

Reply:
this is the `prettier` config file --> https://prettier.io/docs/en/options.html#arrow-function-parentheses